### PR TITLE
Fusion Changes

### PR DESCRIPTION
**Pull request by TechLord22**
This PR makes many changes to the fusion reactors. The structures have changed, new blocks have been added, and recipes have also been changed. The best way to see the differences is in game.

### CODE REVIEW

**Review by htmlcsjs — APPROVED** [review]
looks good from me, might want to force the amount of hatches you need though

**Comment by ALongStringOfNumbers** [comment]
This PR changes the behavior of the fusion formation in a not that great way. Previously you could form the fusion reactor with either LuV, ZPM, or UV energy input hatches (so that you could adjust the total energy pool), and now you are limited to certain tiers of energy input, which means you cannot adjust the total energy pool

**Comment by ALongStringOfNumbers** [comment]
> looks good from me, might want to force the amount of hatches you need though

Forcing the amount of hatches is not really how the fusion reactor is intended to work. You add enough energy inputs to get the the EU total for the recipe that you want to run, with different tiers of energy inputs giving different amounts of energy towards the total pool

File: src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoCategory.java [hunk 1]
```diff
@@ -51,9 +51,9 @@ public MultiblockInfoCategory(IJeiHelpers helpers) {
         put("large_gas_turbine", new MultiblockInfoRecipeWrapper(new LargeTurbineInfo(MetaTileEntities.LARGE_GAS_TURBINE)));
         put("large_plasma_turbine", new MultiblockInfoRecipeWrapper(new LargeTurbineInfo(MetaTileEntities.LARGE_PLASMA_TURBINE)));
         put("assembly_line", new MultiblockInfoRecipeWrapper(new AssemblyLineInfo()));
-        put("fusion_reactor_mk1", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(0)));
-        put("fusion_reactor_mk2", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(1)));
-        put("fusion_reactor_mk3", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(2)));
+        put("fusion_reactor_mk1", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(GTValues.LuV)));
+        put("fusion_reactor_mk2", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(GTValues.ZPM)));
+        put("fusion_reactor_mk3", new MultiblockInfoRecipeWrapper(new FusionReactorInfo(GTValues.UV)));
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoCategory.java:56]: If you are changing these here, you could also change it where the fusion reactors are registered

**Review by ALongStringOfNumbers — APPROVED** [review]
Not really a fan of the glass recipe, but that can always be changed later

**Pull request merged**